draft-ietf-taps-interface.md

```diff
@@ -1054,7 +1054,7 @@ Idempotent is a boolean property. If true, the application-layer entity in the
 Message is safe to send to the remote endpoint more than once for a single
 Send Action. It is used to mark data safe for certain 0-RTT establishment
 techniques, where retransmission of the 0-RTT data may cause the remote
-application to receive the Message multiple times.
+
 
```

Comment:
Is this change intended?

Reply:
+1, this looks like an editor regression

Reply:
Yeah, uh, or fat finger. Oops.